go.mod

```diff
@@ -21,10 +21,10 @@ require (
 	github.com/containers/common v0.62.1
 	github.com/containers/conmon v2.0.20+incompatible
 	github.com/containers/conmon-rs v0.6.6
-	github.com/containers/image/v5 v5.34.2
+	github.com/containers/image/v5 v5.34.3-0.20250314011931-36a971d8ff51
 	github.com/containers/kubensmnt v1.2.0
 	github.com/containers/ocicrypt v1.2.1
-	github.com/containers/storage v1.57.2
+	github.com/containers/storage v1.57.3-0.20250310120440-ab85543c3c6a
```

Comment:
Are we able to stick to a pinned release for those?

Reply:
I updated them because some functions used to mount artifact is only in the latest. Should we wait for the release for those pkgs maybe?

Reply:
No I think that's fine for now on `main`.